sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -995,10 +1006,15 @@ getDeviceLibPrograms(const ContextImplPtr Context, const RT::PiDevice &Device,
     }
 
     bool DeviceSupports = DevExtList.npos != DevExtList.find(ExtStr);
-
     if (!DeviceSupports || InhibitNativeImpl) {
-      Programs.push_back(loadDeviceLibFallback(Context, Ext, Device));
+      Programs.push_back(loadDeviceLibFallback(Context, Ext, Device, false));
```

Comment:
```suggestion
      Programs.push_back(loadDeviceLibFallback(Context, Ext, Device, /*UseNativeLib=*/false));
```
Makes it a little easier to understand what that value is for. 😄
Similar for the call below.

Reply:
Yes, done.